src/invidious/views/post.ecr

```diff
@@ -0,0 +1,31 @@
+<% content_for "header" do %>
+<title>Invidious</title>
+<% end %>
+
+<div id="post" class="comments">
+    <%= IV::Frontend::Comments.template_youtube(post_response.not_nil!, locale, thin_mode) %>
+</div>
+<div id="comments" class="comments">
+</div>
```

Comment:
I've tried the PR on the test instance, and now that I think about it, that'd be great to have a `nojs` option, like on the watch page!

From there, I'm pretty sure that we could finally implement #409 with very few changes!